examples/SPDXYAMLExample-2.2.spdx.yaml

```diff
@@ -158,28 +158,28 @@ packages:
   checksums:
   - algorithm: "MD5"
     checksumValue: "624c1abb3664f4b35547e7c73864ad24"
-  - algorithm: "SHA256"
-    checksumValue: "11b6d3ee554eedf79299905a98f9b9a04e498210b59f15094c916c91d150efcd"
   - algorithm: "SHA1"
     checksumValue: "85ed0817af83a24ad8da68c2b5094de69833983c"
+  - algorithm: "SHA256"
+    checksumValue: "11b6d3ee554eedf79299905a98f9b9a04e498210b59f15094c916c91d150efcd"
   copyrightText: "Copyright 2008-2010 John Smith"
   description: "The GNU C Library defines functions that are specified by the ISO\
     \ C standard, as well as additional features specific to POSIX and other derivatives\
     \ of the Unix operating system, and extensions specific to GNU systems."
   downloadLocation: "http://ftp.gnu.org/gnu/glibc/glibc-ports-2.15.tar.gz"
   externalRefs:
+  - referenceCategory: "SECURITY"
+    referenceLocator: "cpe:2.3:a:pivotal_software:spring_framework:4.1.0:*:*:*:*:*:*:*"
+    referenceType: "http://spdx.org/rdf/references/cpe23Type"
```

Comment:
According to https://github.com/spdx/spdx-spec/issues/451, shouldn't this be just "cpe23Type"?

Reply:
Similar in other places, and also in the JSON.

Reply:
> According to #451, shouldn't this be just "cpe23Type"?

@sschuberth I agree - this is a bug in the Java tools or libraries.  I added an issue to track: https://github.com/spdx/spdx-java-jackson-store/issues/18

Reply:
I just added a commit to manually fix the reference type.

BTW - the reference type for non-SPDX defined types are still full URI's to make sure they are unique.

Reply:
Just FYI - I'll be offline for a few days and may be a bit slow to responding to any additional review comments